<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>JumpDataInfoChoiceDialog</class>
 <widget class="QDialog" name="JumpDataInfoChoiceDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>431</width>
    <height>339</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <family>Segoe UI Semibold</family>
       <pointsize>10</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Jakie informacje o skoku mają być pokazywane?</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QCheckBox" name="checkBox_distance">
         <property name="text">
          <string>Odległość</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox_points">
         <property name="text">
          <string>Nota łączna</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox_gate">
         <property name="text">
          <string>Belka</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox_averagedWind">
         <property name="text">
          <string>Uśredniony wiatr</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox_gateCompensation">
         <property name="text">
          <string>Rekompensata za belkę</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox_windCompensation">
         <property name="text">
          <string>Rekompensata za wiatr</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox_totalCompensation">
         <property name="text">
          <string>Łączna rekompensata</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox_judges">
         <property name="text">
          <string>Noty sędziowskie</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox_judgesPoints">
         <property name="text">
          <string>Punkty od sędziów (suma not)</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QCheckBox" name="checkBox_landingType">
         <property name="text">
          <string>Rodzaj lądowania</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox_specificWind">
         <property name="text">
          <string>Szczegółowy wiatr przy skoku</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox_positionAfterJump">
         <property name="text">
          <string>Miejsce po skoku</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox_jumpPositionInRound">
         <property name="text">
          <string>Pozycja danej noty w danej serii</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox_takeoffRating">
         <property name="text">
          <string>Ocena wybicia</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox_flightRating">
         <property name="text">
          <string>Ocena lotu</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox_landingRating">
         <property name="text">
          <string>Ocena lądowania (zachwianie)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox_snowRain">
         <property name="text">
          <string>Śnieg/Deszcz przy skoku</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_OKButton">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_submit">
       <property name="minimumSize">
        <size>
         <width>120</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>120</width>
         <height>40</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>18</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
border: 1px solid rgb(0, 59, 23);
border-radius: 6px;
color: rgb(0, 0, 0);
background-color: rgb(123, 220, 144);
}
QPushButton:hover{
background-color: rgb(153, 253, 174);
}</string>
       </property>
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
